frontend/src/pages/CarDetailsPage.tsx

```diff
@@ -131,7 +142,7 @@ export default function CarDetailsPage({appUsers, currentUser, updateCarItem, re
                         {editMode? <Button type={"button"}
                                            onClick={() => setEditMode(!editMode)}>Speichern</Button> :
                             <Button type={"submit"}
-                                    onClick={() => onSave()}>Fertig</Button>}
+                                    onClick={() => onSaveNavigate()}>Fertig</Button>}
```

Comment:
```suggestion
                                    onClick={onSaveNavigate}>Fertig</Button>}
```
das hier m. E. n. auch